5.5/Dockerfile

```diff
@@ -50,6 +50,13 @@ ENV PHP_MEMORY_LIMIT=128M \
     PHP_UPLOAD_MAX_FILESIZE=2M \
     PHP_SESSION_SAVE_HANDLER=files \
     PHP_SESSION_SAVE_PATH="" \
-    PHP_MAX_INPUT_VARS=1000
+    PHP_MAX_INPUT_VARS=1000 \
+    PHP_XDEBUG_IDEKEY=PHPSTORM \
+    PHP_XDEBUG_DEFAULT_ENABLE=0 \
+    PHP_XDEBUG_REMOTE_ENABLE=1 \
+    PHP_XDEBUG_REMOTE_AUTOSTART=0 \
+    PHP_XDEBUG_REMOTE_CONNECT_BACK=0 \
+    PHP_XDEBUG_PROFILER_ENABLE=0 \
+    PHP_XDEBUG_REMOTE_HOST=10.254.254.254
 
 WORKDIR /usr/src/app
```

Comment:
you should add a newline to the end of the file to make the red pictogram disappear